src/main/java/com/jossemargt/cookietwist/CookiePot.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Just curious about why is not enough adding the MIT license on the repo. I mean, why do you need to add it on all your classes?

Reply:
I have to honest with you, I just followed the patterns that I have observed in other open source Java projects, like Jenkins and several Jenkins plugins. By the way I did a quick google search about the topic and the tl;dr answer is "to remind about the license and that any extract of the source code is tied to it". 

So I'll stick to it, since those lines can be ignored and does not alter the artifact nor the javadoc :man_shrugging: